product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,265 @@
+---
+title: Step 4 - Installing the PGD software
+navTitle: Installing PGD software
+deepToC: true
+---
+
+## Installing the PGD software
+
+With the repositories configured, you can now install the Postgres Distributed software. 
+These steps must be carried out on each host.
+
+* Install the PGD packages which include a server specific BDR package and generic PGD proxy and cli packages.
+* Initialise the Postgres database.
+* Add the BDR extension to the shared_preload_libraries setting in `postgresql.conf`.
+* Set the `wal_level` GUC variable to `logical` in `postgresql.conf`.
+* Turn on commit timestamp tracking by setting `track_commit_timestamp` to `'on'` in `postgresql.conf`.
+* Set a password on the EnterprisedDB/Postgres user.
+* Raise the maximum worker processes to 16 by setting `max_worker_processes` to `'16'` in `postgresql.conf`.
+* Add rules to `pg_hba.conf` to allow nodes to connect to each other.
+* Add a `.pgpass` file to allow nodes to authenticate each other.
+* Restart the server.
+* Log into the server's default database (`ebd` for EPAS, `postgres` for PGE and Community).
+* Use `CREATE DATABASE bdrdb` to create the default PGD replicated database.
+* Log out and then log back in to `bdrdb`.
+* Use `CREATE EXTENSION bdr` to enable the BDR extension and PGD to run on that database.
+
+We will look in detail the steps for EDB Postgres Advanced Server in the worked example below. 
+
+If you are installing PGD with EDB Postgres Extended Server or Community Postgres, the steps are similar, but details such as package names and paths are different. These differences are summarized in [Installing PGD for EDB Postgres Extended Server](#installing_pgd_for_edb_postgres_extended_server) and [Installing PGD for Community Postgresql](#installing_pgd_for_community_postgresql).
+
+## Worked example
+
+### Install the packages
+
+The first step is to install the packages. For each Postgres package, there is a `edb-bdr5-<postgresversion>` package to go with it. 
+For example, if we are installing EDB Postgres Advanced Server (epas) version 16, we would install `edb-bdr5-epas16`. 
+
+There are two other packages to also install: `edb-pgd5-proxy` for PGD Proxy and `edb-pgd5-cli` for the PGD command line tool.
```

Comment:
This would be more readable as a short list, i.e.

```
There are two other packages to also install:

- `edb-pgd5-proxy` for PGD Proxy
- `edb-pgd5-cli` for the PGD command line tool

```

(When reading docs which enumerate items, I find it _much_ easier to scan when they're noted as a list, rather than a run-on sentence -> putting my accepted doc patches where my mouth is, see https://git.postgresql.org/gitweb/?p=postgresql.git;a=commitdiff;h=f246c48ac4ac323152d3569bed1ccd571a5cc3d8  :) )

Reply:
Fixed and noted.